### ISSUE TITLE
Enable TLS for HTTP1 protocol

### ISSUE THREAD

**Issue by nluaces** [skupperproject/skupper#1062]
## Summary
Enable the skupper to export or create services over TLS using the HTTP1 protocol. 

  
## Rationale

The skupper router now allows using TLS with the HTTP1 adaptor, so it is required that the control plane enables this functionality.

It would be necessary to remove the restrictions regarding the protocol and to make sure that the sslProfiles references in the router connectors/listeners are configured.
  

## Testing plan 

- [x] Manual test: After exposing a service with TLS option and HTTP1 protocol, check if the connectors and listeners are created correctly in all the routers of the VAN.
- [x] Integration test: set up an nginx server, configure the certificates and send a request expecting a 200 http status response.

  
## Documentation plan
At present, the TLS option was available only for HTTP2 and TCP protocols. All the documentation that references this restriction should be changed after merging the solution.

In other words, there will be no restrictions in protocols when using TLS reencryption.
